tasks/gulp/copy-to-destination.js

```diff
@@ -25,5 +30,24 @@ gulp.task('copy-files', () => {
       autoprefixer
     ], {syntax: require('postcss-scss')}))
     .pipe(scssFiles.restore)
+    .pipe(yamlFiles)
+    .pipe(map(function (file, done) {
+      let componentName = path.dirname(file.path).split(path.sep).slice(-1).toString()
+      let componentPath = path.join(configPaths.components, componentName, `${componentName}.yaml`)
+
+      try {
+        let json = yaml.safeLoad(fs.readFileSync(componentPath), {encoding: 'utf8', json: true})
```

Comment:
the `readFileSync` could be pulled out into a line above to make it more readable and have fewer operations happening in a single line

Reply:
Changed!